src/licensedcode/detection.py

```diff
@@ -1356,6 +1375,53 @@ def has_references_to_local_files(license_matches):
     )
 
 
+def use_referenced_license_expression(referenced_license_expression, license_detection, licensing=Licensing()):
```

Comment:
What about this:
```
    Return True if the ``license_detection`` LicenseDetection should include
    the matches represented by the ``referenced_license_expression`` string.
    Return False otherwise.
    This determination is based on multiple factors detailed in comments below. 
    
    Used when we have a ``license_detection`` with a match to a license rule like
    "See license in COPYING" and where the ``referenced_license_expression`` is the
    expression found in the "COPYING" file. This  expression is the combined
    expression for all matches found in "COPYING".

    The actual combination of other matches in ``license_detection`` is the
    responsibility of the caller.
```

Reply:
Done, thanks!